core/webengine.py

```diff
@@ -249,6 +254,17 @@ def eventFilter(self, obj, event):
                 event.accept()
                 return True
 
+            elif event.button() == MOUSE_LEFT_BUTTON:
+                modifiers = QApplication.keyboardModifiers()
+                if modifiers == Qt.ControlModifier:
+                    self.open_url_new_buffer_other_window(self.url_hovered)
+                    return True
+
+            elif event.button() == MOUSE_WHEEL_BUTTON:
```

Comment:
Please add Ctrl modifier check, if only check wheel event, user will trigger open new window when he scroll page after hover something.

Reply:
Hmmm, I don't encounter this problem. I can scroll normally when hovering over the link. Only when the WheelButton is pressed, then the link will be opened (this is similar to Google Chrome's feature).

Can you double-check if this problem occurs in your machine?